actionpack/lib/action_dispatch/middleware/api_public_exceptions.rb

```diff
@@ -0,0 +1,47 @@
+module ActionDispatch
+  class ApiPublicExceptions
```

Comment:
@rafaelfranca great catches bro. Yeah this was needed some time ago but it is not anymore.
Removed https://github.com/spastorino/rails/commit/c23d78e783cc88d78f1b8990d88fa76ff164096c